web/core/components/workspace/sidebar/favorites/favorite-folder.tsx

```diff
@@ -101,22 +89,36 @@ export const FavoriteFolder: React.FC<Props> = (props) => {
       draggable({
         element,
         getInitialData: () => initialData,
-        onDragStart: () => setIsDragging(true),
+        // onDragStart: () => setIsDragging(true),
```

Comment:
Is this intentional? if so any reason for this change

Reply:
Added it to remove disable opacity to a folder on add. Changed the approach now. Using onDrag instead of onDragStart.